plugins/win-mf/mf-plugin.cpp

```diff
@@ -30,6 +30,10 @@ extern "C" void obs_module_unload(void)
 }
 
 OBS_DECLARE_MODULE()
+MODULE_EXPORT const char *obs_module_description(void)
+{
+	return "Windows Media Foundations H.264/AAC encoder";
```

Comment:
I thought WMF was only used for audio.  Does AAC have to be designated as "H.264/AAC"?